contentcuration/contentcuration/frontend/channelEdit/vuex/contentNode/getters.js

```diff
@@ -277,7 +299,9 @@ export function copyrightHolders(state) {
 }
 
 export function tags(state) {
-  return uniq(flatMap(Object.values(state.contentNodesMap), node => node['tags']).filter(t => t));
+  return uniq(
+    flatMap(Object.values(state.contentNodesMap), node => Object.keys(node['tags'])).filter(t => t)
```

Comment:
I'm getting `vue.runtime.esm.js:1888 TypeError: Cannot convert undefined or null to object` here when I go to make a new subtopic

Reply:
This should now be fixed.